hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/protocol/DatanodeDetails.java

```diff
@@ -619,11 +629,11 @@ public void setCurrentVersion(int currentVersion) {
 
   @Override
   public String toString() {
-    return uuidString + "(" + hostName + "/" + ipAddress + ")";
+    return id + "(" + hostName + "/" + ipAddress + ")";
   }
 
   public String toDebugString() {
-    return uuid.toString() + "{" +
+    return id.toString() + "{" +
```

Comment:
nit: can drop explicit `toString()`

```suggestion
    return id + "{" +
```